gkeys/gkeys/mail.py

```diff
@@ -37,10 +38,11 @@ def __init__(self, login, logger):
     def send_email(self, uid, message):
         self.logger.debug(_unicode('Sending email with message %s') % _unicode(message))
         subject = "Expiring Key"
-        email_to = _unicode("justthisthing@gmail.com") #uid
+        email_to = uid
         msg = MIMEText(message, 'plain')
         msg['Subject'] = subject
-        msg['From'] = self.email_from
+        msg['From'] = email.utils.formataddr((self.sender_full_name, self.email_from))
+        msg['To'] = email_to
         self.logger.info(_unicode('Sending the email reminder from %s to %s') \
             % (self.email_from, email_to))
         self.mail.sendmail(self.email_from, email_to, msg.as_string())
```

Comment:
All commits up to this point should be squashed together, then possibly broken apart again partially for a few commits for various feature changes.  But as it is, they are not merge worthy due to all the hard-coding of config values that should not be part of the gentoo-keys code base.  They are fine in Angelos repository for his development history.  But they do not belong in the main gentoo-keys repository.  So merging all the previous commits into one is a minimum requirement for this to be merged.